dotnet/samples/KernelSyntaxExamples/Getting_Started/Step9_Safe_Chat_Prompts.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Examples;
+using Microsoft.SemanticKernel;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace GettingStarted;
+
+public sealed class Step9_Safe_Chat_Prompts(ITestOutputHelper output) : BaseTest(output)
+{
+    /// <summary>
+    /// Show how to construct a chat prompt safely and invoke it.
+    /// </summary>
+    [Fact]
+    public async Task RunAsync()
+    {
+        // Create a logging handler to output HTTP requests and responses
+        var handler = new LoggingHandler(new HttpClientHandler(), this.Output);
+        var client = new HttpClient(handler);
+
+        // Create a kernel with OpenAI chat completion
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddOpenAIChatCompletion(
+                modelId: TestConfiguration.OpenAI.ChatModelId,
+                apiKey: TestConfiguration.OpenAI.ApiKey,
+                httpClient: client)
+            .Build();
+
+        // Each example demonstrates a different way to construct a chat prompt
+        await ExamplePlainTextAsync(kernel);
```

Comment:
Can we remove this method and decompose it in each example to be its own fact and moving  the Kernel creation to a private method?

Reply:
Yes, I have a follow up task to do that: https://github.com/microsoft/semantic-kernel/issues/5881